about.html

```diff
@@ -1,84 +1,116 @@
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class='navbar-brand' id='logo-img' href='index.html'><img src='images/logo.png' title='Macoupin Resource Directory' alt='Macoupin Resource Directory' style='height: 40px;'/></a>
-        </div>
-        <div class="navbar-collapse collapse">
-          <ul class="nav navbar-nav navbar-right">
-            <li><a href="index.html"><i class='fa fa-map-marker'></i> Map</a></li>
-            <li class='active'><a href="about.html"><i class='fa fa-info-circle'></i> About</a></li>
-            <li><a href='https://www.macoupinvotes.com/machform/view.php?id=11280'><i class='fa fa-plus-circle'></i> Add Your Information</a></li>
-            <li><a href='http://macoupincountyil.gov/county_clerk.htm'>County Clerk</a></li>
-          </ul>
-        </div><!--/.nav-collapse -->
+
+    <nav class="navbar navbar-expand-md navbar-light bg-light mb-2">
+      <a class="navbar-brand" href="/"><img src='/images/logo.png' alt="Macoupin Resource Directory" title="Macoupin Resource Directory" style='height: 50px;'/></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse mr-auto" id="navbarsExampleDefault">
+        <ul class="navbar-nav ml-auto">
+          <li class='nav-item'>
+            <a class="nav-link" href="/"><i class="fa fa-map-marker" aria-hidden="true"></i> Map</a>
+          </li>
+          <li class='nav-item'>
+            <a class="nav-link active" href="/about.html"><i class="fa fa-info-circle" aria-hidden="true"></i> About</a>
+          </li>
+          <li class='nav-item'>
+            <a class="nav-link" href="https://www.macoupinvotes.com/machform/view.php?id=11280"><i class="fa fa-plus-circle" aria-hidden="true"></i> Add your information</a>
+          </li>
+          <li class='nav-item'>
+            <a class="nav-link" href="http://macoupincountyil.gov/county_clerk.htm">County Clerk</a>
+          </li>
+        </ul>
       </div>
-    </div>
-    <div class='container-fluid'>
-      <div class='row'>
-        <div class='col-md-8'>
-          <h1>About</h1>
-          
-          <p>Macoupin Directory serves as Macoupin County’s online source for finding businesses, local governments and community services across the county.</p>
+    </nav>
+
+    <main role="main" class="container-fluid">
+
+      <div class="container">
+        <div class="row">
+          <div class="col-md-8">
+
+            <div class='row'>
+              <div class='col-md-12'>
+                <h3 id="about"><i class="fa fa-info-circle" aria-hidden="true"></i> About</h3>
```

Comment:
Looks like Bootstrap Reboot [resets all heading elements to have no `margin-top`](https://getbootstrap.com/docs/4.0/content/reboot/#headings-and-paragraphs), which causes the first header on this page to look a little squished with the navbar:

<img width="877" alt="Screen Shot 2019-11-15 at 9 37 14 AM" src="https://user-images.githubusercontent.com/14170650/68955424-9b905f00-078b-11ea-8860-acdebf30368a.png">

I would add a small margin to give the heading some breathing room:

```suggestion
                <h3 class="mt-4" id="about"><i class="fa fa-info-circle" aria-hidden="true"></i> About</h3>
```